From ee0854e2a269b4b3388fc5a1d00fb19359b2f9e9 Mon Sep 17 00:00:00 2001
From: Michael Cronenworth <mike@cchtml.com>
Date: Sun, 13 Jul 2025 14:30:13 -0500
Subject: [PATCH] fix: API URL changed

One person says the my.homerun.com domain name was deprecated years ago.
SiliconDust terminated it July 12, 2025.
---
 src/HDHomeRunTuners.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/HDHomeRunTuners.cpp b/src/HDHomeRunTuners.cpp
index 0e1f1ff..121de87 100644
--- a/src/HDHomeRunTuners.cpp
+++ b/src/HDHomeRunTuners.cpp
@@ -238,7 +238,7 @@ bool HDHomeRunTuners::Update(int nMode)
     //
     if (nMode & UpdateGuide)
     {
-      strUrl = kodi::tools::StringUtils::Format("https://my.hdhomerun.com/api/guide.php?DeviceAuth=%s", EncodeURL(pTuner->Device.device_auth).c_str());
+      strUrl = kodi::tools::StringUtils::Format("https://api.hdhomerun.com/api/guide.php?DeviceAuth=%s", EncodeURL(pTuner->Device.device_auth).c_str());
       KODI_LOG(ADDON_LOG_DEBUG, "Requesting HDHomeRun guide: %s", strUrl.c_str());
 
       if (GetFileContents(strUrl.c_str(), strJson))
-- 
2.50.0

